rskj-ubuntu-installer/config/regtest.conf

```diff
@@ -273,28 +90,5 @@ miner {
 
     # this is a secret passphrase that is used to derive the address where the miner gets the reward.
     # please note this is stored in a local wallet and not recommended for production.
-    coinbase.secret = <COINBASE_SECRET>
```

Comment:
miner server is enabled by default in regtest. you could simplify to `miner.coinbase.secret = <COINBASE_SECRET>`